jax/experimental/jax2tf/jax2tf.py

```diff
@@ -831,8 +839,15 @@ def _convert_value(val, aval):
   kept_args_avals = [aval for i, aval in enumerate(exported.in_avals) if i in exported.module_kept_var_idx]
   kept_args_flat_tf = [atf for i, atf in enumerate(args_flat_tf) if i in exported.module_kept_var_idx]
 
+  if hasattr(tfxla, "call_module_maximum_supported_version"):
+    max_version_supported = tfxla.call_module_maximum_supported_version()
```

Comment:
In open source we have to handle the case when different packages work at different versions. Recently we added `tfxla.call_module_maximum_supported_version` so that front-ends can customize the attributes they pass based on the version supported; alternatively, the front-end can raise an error and ask the user to upgrade tensorflow.

My plan is the following. `XlaCallModule` will have a minimum and a maximum supported version, to cover a 6 month compatibility guarantee. There will be an error when running ops at versions outside this range. `jax_export.export` will always produce an `Exported` at the maximum version supported by the front-end. Today, `Exported` has an attribute `xla_call_module_version`. This will disappear, and instead the version will be a constant that changes only when you upgrade JAX. (If in the future we will add loading of `Exported`, they will be upgraded to the current version supported by the library, or there will be a loading error.) When the front-end tries to emit the `XlaCallModule`, it will check the maximum version supported by the current version of TF and will try to "downgrade" the `Exported`. If it is not possible there will be an error.

At present, the latest version of `Exported` is 6, and if the `XlaCallModule` does not support version 6, it is possible to "downgrade" by not adding the `disabled_checks` attribute.

Reply:
Thanks a lot for your explain